src/SFML.Graphics/Text.cs

```diff
@@ -97,18 +97,20 @@ public enum Styles
 
         ////////////////////////////////////////////////////////////
         /// <summary>
-        /// Fill color of the object
+        /// DEPRECATED <see cref="SFML.Graphics.Color"/> of the <see cref="Text"/>
```

Comment:
The `Fill` was dropped

Reply:
I'll add a bit more clarity to this.  People should be moving away from `Text.Color` though, rather than getting better documentation for it. 🤣